lib/core-integration/src/Test/Integration/Framework/Request.hs

```diff
@@ -82,6 +82,9 @@ data Context t = Context
     , _port
         :: Port
         -- ^ Server TCP port
+    , _jormungandrBaseUrl
+        :: Text
+        -- ^ Jormungandr REST API port
```

Comment:
Let's rename this to `nodePort` since:

- Jormungandr doesn't exist in this realm
- We only store a port in here, and not an actual URL (I believe this is the result of some successive changes).